receiver/RatingReceiver.py

```diff
@@ -0,0 +1,12 @@
+import asyncio
+import websockets
+import json
+
+async def listen_for_new_ratings():
+    async with websockets.connect("ws://localhost:8765") as websocket:
```

Comment:
Leaving hard coded strings like that (`"ws://localhost:8765"`) is a bad idea. 
Replace this with an environment variable or a `const` from a `consts` file.

Reply:
On it.

Reply:
Moved to .env file.
 - provided example.env for reference.